general/releases/changelogs/0.20.0/conversion.md

```diff
@@ -0,0 +1,102 @@
+
+The following sections describe [how to convert an asset with a `RenderableBillboardsCloud` to the new format](./conversion.md#how-to-update-an-asset-with-a-renderablebillboardscloud), or how to [update a color map file](./conversion.md#update-color-map-data-format) (the .cmap data format has changed slightly). Finally, there is also a [summary of the updated property names](./conversion.md#summary-of-updated-properties), that can hopefully be useful if you have Lua scripts or actions that require updating due to the use properties that have been updated or removed.
+
+:::{note}
+Note that the previously existing `RenderablePoints` renderable has been removed in favor of the new `RenderablePointsCloud`.
+:::
+
+### How-to Update an Asset with a `RenderableBillboardsCloud`
+Below is a code snippet showing an example of parts of an asset with a `RenderableBillboardsCloud`, before and after the update.
+The highlighted lines show the fields whose format or name has changed. See comments in the "After" example below for details.
+
+::::{tab-set}
+:::{tab-item} Before
+```{literalinclude} files/pointcloud_before.lua
+:language: lua
+:emphasize-lines: 4, 11-14, 16, 18-20
+```
+:::
+
+:::{tab-item} After
+```{literalinclude} files/pointcloud_after.lua
+:language: lua
+:emphasize-lines: 4, 11-25, 27-30, 32-36
+```
+:::
+::::
+
+:::{tip}
+Note that the `Texture` field is no longer required. If a texture is excluded, the points will be drawn as circles.
+:::
+
+#### Compute `ScaleExponent` from `ScaleFactor`
+The scaling of the points has been made more intuitive by changing the previous `ScaleFactor` property slightly, to make it more apparent that it affects an exponential scaling. See [details on the point data page](/manual/content/point-data/point-data.md#controlling-the-point-size), but in general, with the new `ScaleExponent` the value is used as the exponent for `10^exponent` to compute the world-scale size of the points.
+
+The new `ScaleExponent` property can be computed from the previous `ScaleFactor` as:
+:::{math}
+ScaleExponent = \log_{10}(e^{ScaleFactor / 10})
+:::
+
+### Update Color Map Data Format
+
+Previously, the first and last color entry in a .cmap file was implicitly interpreted as the color to use for values that are lower or higher than the specified range. However, this was not very clear and has been made more explicit in the update by using specific keywords for these colors. Here is an example of a color map before and after this change:
+
+::::{tab-set}
+:::{tab-item} Before
+```{literalinclude} files/before.cmap
```

Comment:
Maybe name this colormap_before.cmap?  Even though it is obvious it might be easier to understand when later looking at the code